Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource/helper.go

```diff
@@ -117,7 +117,7 @@ func (m *Helper) Create(namespace string, modify bool, data []byte) (runtime.Obj
 			if err := m.Versioner.SetResourceVersion(obj, ""); err != nil {
 				return nil, err
 			}
-			newData, err := m.Codec.Encode(obj)
+			newData, err := m.RESTClient.GetCodec().Encode(obj)
```

Comment:
> I just don't like adding GetCodec() to the RESTClient interface... passing the object into Body lets you leave the responsibility for encoding in the RESTClient's hands

I don't like the sig `func (m *Helper) Create(namespace string, modify bool, data []byte)` either, but changing it to `func (m *Helper) Create(namespace string, modify bool, data interface{})` is more that I'd really like to change here